src/wp-includes/blocks.php

```diff
@@ -108,12 +108,20 @@ function register_block_script_handle( $metadata, $field_name ) {
 	}
 	// Path needs to be normalized to work in Windows env.
 	$wpinc_path_norm  = wp_normalize_path( realpath( ABSPATH . WPINC ) );
+	$theme_path_norm  = wp_normalize_path( get_theme_file_path() );
```

Comment:
Let’s apply `realpath` for this path as well following other cases. It prevents some mismatches on Windows platform. Unless it’s already the case for `get_theme_file_path`. I couldn’t locate it in the code.

Reply:
I've added the `realpath` wrapper to both instances of the `$theme_path_norm` variable in https://github.com/WordPress/wordpress-develop/pull/2494/commits/679eee0bd0df9553fa72a4da4892e938d488d290